src/components/rails/explorer-rail/explorer.vue

```diff
@@ -84,7 +84,26 @@ export default {
     </div>
     <div class="node-types__container" v-if="tabIndex === 0">
       <filter-node-types />
-      <node-types-loop />
+      <template v-if="filteredNodes.length > 0">
+        <node-types-loop
+          v-on="$listeners"
+          :nodeTypes="filteredNodes"
+        />
+      </template>
+      <template v-else>
+        <node-types-loop
+          v-if="pinnedObjects.length > 0"
+          v-on="$listeners"
+          label="Pinned Objects"
+          :nodeTypes="pinnedObjects"
+        />
+        <node-types-loop
+          v-if="objects.length > 0"
+          v-on="$listeners"
+          label="Object Category"
+          :nodeTypes="objects"
+        />
+      </template>
```

Comment:
uhm I believe these lines have changed since a while ago. I believe I refactored the node-types-loop